### ISSUE TITLE
It's not an issue, just a question in mind

### ISSUE THREAD

**Issue by feldrim** [danielquinn/paperless#108]
Why should I scan the document into pdf which is generally not in the standardized format in the specification (ISO 32000-1:2008) due to many programmers' practical(!) "save as pdf" solution? Wouldn't it be ok if the documents scanned and exported into a lossless or nearly lossless image format? 

This is an obviously a conscious decision. I just wonder what are the advantages & disadvantages in comparison.

**Comment by JaimeObregon** [comment]
Hi! There are a number of problems with storing the document corpus in image files, mainly the need to code additional logic to handle multi-page documents, cryptography, annotations or metadata in image formats.

**Comment by feldrim** [comment]
Oh, I see. And in the end you have to save a searchable pdf in the server. I was just thinking about shortening the conversion process written in README. By the way, is this [OCRmyPDF library ](https://github.com/jbarlow83/OCRmyPDF) considered?

**Comment by danielquinn** [comment]
Everything @JaimeObregon said is correct, but I should also point out that Paperless supports PDF, JPG, PNG, GIF, and TIFF files, so if you're starting with these file types at the start, you don't need to convert them to PDF to index them with Paperless.

As for the OCRmyPDF library, it looks pretty handy, but I'm not thrilled about the idea of modifying PDFs that are consumed.  Currently, we do all kinds of image manipulation stuff to get a good OCR value, but the software doesn't make any edits to the original before it stores it in the media directory.  OCRmyPDF appears to modify the document directly, so if I were to use it, I'd have to introduce some sort of option to allow users to opt-out before I mangle their docs.

So TL;DR: seems like a cool idea, but I'll have to spend a little time thinking about how to do this if I do it at all.
